docs/get-started/lotus/installation.md

```diff
@@ -76,9 +80,9 @@ You can find out more about this Snap [over at Snapcraft.io](https://snapcraft.i
     ./lotus-v1.10.0_linx-amd64.appimage
     ```
 
-## Build from source
+### Build from source
```

Comment:
Thanks for the header level fixes, this is correct logically. Before merging, please check that this doesn't break sidebar formatting in the netlify preview.

Reply:
Sidebar looks good.